diff --git a/config.def.h b/config.def.h
index 1edb647..3fe0a91 100644
--- a/config.def.h
+++ b/config.def.h
@@ -9,8 +9,8 @@ static const char *fonts[] = {
 static const char *prompt      = NULL;      /* -p  option; prompt to the left of input field */
 static const char *colors[SchemeLast][2] = {
 	/*     fg         bg       */
-	[SchemeNorm] = { "#bbbbbb", "#222222" },
-	[SchemeSel] = { "#eeeeee", "#005577" },
+	[SchemeNorm] = { "#c0caf5", "#1a1b26" },
+	[SchemeSel] = { "#c0caf5", "#414868" },
 	[SchemeOut] = { "#000000", "#00ffff" },
 };
 /* -l option; if nonzero, dmenu uses vertical list with given number of lines */
